etk/extractors/language_identification_extractor.py

```diff
@@ -33,4 +37,25 @@ def extract(self, text: str, method: str) -> List[Extraction]:
         Returns the empty list of the extractor fails to identify the language in the text.
 
         """
-        pass
+        if method == IdentificationTool.LANGID.name:
+            language = classify(text)[0]
+            return [Extraction(value=language, extractor_name=self.name)]
+
+        elif method == IdentificationTool.LANGDETECT.name:
+            try:
```

Comment:
```
 try:
+                language = detect(text)
+            except:
+                language = 'unknown'
+
+            if language == 'unknown':
+                return list()
```
This whole block of code can be replaced as,
```
 try:
                 language = detect(text)
             except:
                 return list()
```

Reply:
Langdetect may return an string 'unknown' , so I separate them